README.md

```diff
@@ -19,6 +22,26 @@ $ cd git-machete
 $ sudo make install
 ```
 
+### Using setup
+
+Run the following commands to install git machete:
+
+```bash
+$ git clone https://github.com/VirtusLab/git-machete.git
+$ cd git-machete
+$ python setup.py install
+```
+
+To install you need to have Python with installed `pip` and `setuptools`. Recommended Python env is installed with [Pyenv](https://github.com/pyenv/pyenv).
+
+If you use system Python env, you will need to execute last command with `sudo` or with option `--user`:
```

Comment:
I guess sudo will be needed even in case of Python coming from `pyenv` b/c of the Bash completion script, right?

Reply:
I found solution for that, it will be described `README.md` in commit https://github.com/VirtusLab/git-machete/pull/13/commits/b2fa5a66c1dc322352196a55c89fd87762e82100.